src/router/redirect.js

```diff
@@ -14,7 +14,7 @@ export const logOutAndRedirectToLogin = async (params) => {
   store.dispatch('userProfile/resetUserProfile');
   store.dispatch('planning/resetPlanning');
 
-  if (router.currentRoute.path === '/login') return;
+  if (router.currentRoute.value.path === '/login') return;
```

Comment:
`this.$route.path`

Reply:
on n'est pas dans un composant la donc on ne peut pas utiliser `this.$route`